worksheet_A/TerminalHacking.cpp

```diff
@@ -3,9 +3,32 @@
 
 #include "stdafx.h"
 #include "WordList.h"
+#include <string>
+#include <iostream>
```

Comment:
Better to add these to `stdafx.h` to take advantage of header precompilation (i.e. faster compile times).